src/views/Delegate.vue

```diff
@@ -198,6 +255,28 @@ onMounted(async () => {
             />
           </div>
         </Block>
+        <Block
```

Comment:
This block should show as loading, and once it's loaded it should only show if `delegatesWithScore.length > 0`

Reply:
@gbonumore you need to add :loading="delegatesLoading" to the block.